enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -1009,11 +1008,9 @@
                          (mt/rows (mt/run-mbql-query orders {:limit 1})))))))))))))
 
 (deftest pivot-query-test
-  (mt/test-drivers (disj
-                    (mt/normal-drivers-with-feature :foreign-keys :nested-queries :left-join)
-                    ;; this test relies on a FK relation between $product_id->products.category, so skip for Presto
-                    ;; JDBC, because that driver doesn't support resolving FKs from the JDBC metadata
-                    :presto-jdbc)
+  ;; This test relies on a FK relation between $product_id->products.category, hence the requirement on
+  ;; :metadata/key-constraints.
+  (mt/test-drivers (mt/normal-drivers-with-feature :foreign-keys :nested-queries :left-join :metadata/key-constraints)
     (testing "Pivot table queries should work with sandboxed users (#14969)"
```

Comment:
FWIW we should still be able to test this for BigQuery and the other drivers if we wrap this in `mt/with-mock-fks-for-drivers-without-fk-constraints`, altho it would be even better if we started doing it automatically when we load test data of course. Fixing that is in my mental TODO list

Reply:
Ah, interesting, I didn't know about that wrapper.

Reply:
I opened https://github.com/metabase/metabase/issues/45381 to track doing that stuff automatically so we don't need that wrapper anymore and can run all the tests without having to worry about `:metadata/key-constraints` or whatever

Reply:
@camsaul, that issue will get solved by https://github.com/metabase/metabase/pull/44894, where I'm addressing the review remarks now.

@bshepherdson , `mt/with-mock-fks-for-drivers-without-fk-constraints` will be removed by my PR. I assume this will get in earlier, so I'll handle the `:metadata/key-contraints` vs `:foreign-keys` here in conflicts resolution.

Reply:
oh nice!